src/groups/bmq/bmqeval/bmqeval_simpleevaluator.t.cpp

```diff
@@ -520,6 +521,14 @@ static void test3_evaluation()
         {"i_0 != -9223372036854775807", true},  // -(2 ** 63) + 1
         {"i_0 != 9223372036854775807", true},   // 2 ** 63 - 1
         {"i_0 != -9223372036854775808", true},  // -(2 ** 63)
+
+        // exists
+        {"exists(i_42)", true},
+        {"exists(non_existing_property)", false},
```

Comment:
Also the interesting case when either a property doesn't exist or has a constrained value:
```suggestion
        {"exists(non_existing_property)", false},
        {"!exists(non_existing_property) || non_existing_property > 41", true},
```